py/server/deephaven/table_factory.py

```diff
@@ -373,14 +373,14 @@ def function_generated_table(table_generator: Callable[[], Table],
         refresh_interval_ms (int): Interval (in milliseconds) at which the 'table_generator' function is rerun.
         exec_ctx (ExecutionContext) -> Table): A custom execution context. If 'None', the current
```

Comment:
Probably a mistake we missed when it was first added.

Reply:
Should it be
```
def function_generated_table(table_generator: Callable,
           source_tables: Union[Table, List[Table]] = None,
           refresh_interval_ms: int = None,
           exec_ctx: ExecutionContext = None,
           **kwargs) -> Table:
```